modules/layers/src/path-layer/path.js

```diff
@@ -0,0 +1,19 @@
+import {cutPolylineByGrid} from '@math.gl/polygon';
+
+// Returns: a flat array of path positions
```

Comment:
Update comment? as this function does a bit more than flatten. Mention that it the path is also being cut? Generally some TLC on the comments would be nice. Using `/**  Returns a flat array of path positions */` syntax is also nice in vscode as it becomes available as intellisense on hover.

As a total nit: Generally I think we are now overusing the word `normalize...` it usually connotes a rather light mainly structural operation to clean up some edge cases, but in our code based it is increasingly being used as a synonym for `preprocess...` :)